apps/docs/pages/_app.tsx

```diff
@@ -68,28 +70,32 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
         <SessionContextProvider supabaseClient={supabase}>
           <AuthProvider>
             <ThemeProvider>
-              <SearchProvider>
-                {/* <CommandMenuProvider site="docs"> */}
+              <CommandMenuProvider
+                site="docs"
+                MarkdownHandler={({ ...props }) => (
+                  <ReactMarkdown remarkPlugins={[remarkGfm]} components={components} {...props} />
+                )}
+              >
                 <SiteLayout>
                   <Component {...pageProps} />
                 </SiteLayout>
-                {/* </CommandMenuProvider> */}
-              </SearchProvider>
+              </CommandMenuProvider>
             </ThemeProvider>
           </AuthProvider>
         </SessionContextProvider>
       ) : (
-        <AuthProvider>
```

Comment:
@alaister do we need the AuthProvider here? i remember in your previous PR we declared it here